#
# old_revision [7950c8a4e13342d1dca94c05efc49b291ab327f5]
#
# patch "a"
#  from [023cc93272e5cbc11f128ffc90fbbe42f6f08d8c]
#    to [b3d40c241ccfd6ff1a06047a9f6d609227d2893f]
#
============================================================
--- a	023cc93272e5cbc11f128ffc90fbbe42f6f08d8c
+++ a	b3d40c241ccfd6ff1a06047a9f6d609227d2893f
@@ -1 +1,2 @@ ls
 ls
+ls
